### ISSUE TITLE
v0.4.0: installation fails

### ISSUE THREAD

**Issue by JayBrown** [derhuerst/vbb-stations-cli#2]
Tried updating from v0.3.1 to v0.4.0, which didn't work; instead uninstalled & tried to reinstall. Didn't work either:

```
 ❯ npm install -g vbb-stations-cli                                                                                               [11:18:14]
npm ERR! Darwin 15.4.0
npm ERR! argv "/usr/local/Cellar/node/6.1.0/bin/node" "/usr/local/bin/npm" "install" "-g" "vbb-stations-cli"
npm ERR! node v6.1.0
npm ERR! npm  v3.8.6
npm ERR! path /usr/local/lib/node_modules/vbb-stations-cli/bin.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod '/usr/local/lib/node_modules/vbb-stations-cli/bin.js'
npm ERR! enoent ENOENT: no such file or directory, chmod '/usr/local/lib/node_modules/vbb-stations-cli/bin.js'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! Please include the following file with any support request:
npm ERR!     ~/npm-debug.log
```

Note: npm didn't save any npm-debug.log file.

**Comment by JayBrown** [comment]
Tried an installation without the -g option, i.e. into the user directory, and that did produce an npm-debug.log file. Here it is:
[npm-debug.log.zip](https://github.com/derhuerst/vbb-stations-cli/files/256830/npm-debug.log.zip)

**Comment by derhuerst** [comment]
I pushed `0.4.1` which should works, thanks.

The issue however was a really weird one. `bin.js` was missing in the `vbb-stations-cli@0.4.0`, even though the `files` field in `package.json` has already been correct _before_ `0.4.0`.

**Comment by JayBrown** [comment]
Same error again with v0.7.0

```
 ❯ npm install -g vbb-stations-cli                                                                                               [16:40:01]
npm ERR! Darwin 15.5.0
npm ERR! argv "/usr/local/Cellar/node/6.3.0/bin/node" "/usr/local/bin/npm" "install" "-g" "vbb-stations-cli"
npm ERR! node v6.3.0
npm ERR! npm  v3.10.3
npm ERR! path /usr/local/lib/node_modules/vbb-stations-cli/bin.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod

npm ERR! enoent ENOENT: no such file or directory, chmod '/usr/local/lib/node_modules/vbb-stations-cli/bin.js'
npm ERR! enoent ENOENT: no such file or directory, chmod '/usr/local/lib/node_modules/vbb-stations-cli/bin.js'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! Please include the followi
ng file with any support request:
npm ERR!     ~/npm-debug.log
>>> elapsed time 21s
```

[npm-debug.log.zip](https://github.com/derhuerst/vbb-stations-cli/files/357348/npm-debug.log.zip)

**Comment by derhuerst** [comment]
npm/npm#5082 again! :sob:

Fixed with `vbb-stations-cli@0.7.1`.
